README.md

```diff
@@ -88,4 +88,18 @@ If you install Vagrant, you can run the application in its own vm. Just install
 ```bash
 $ vagrant up
 
-```
\ No newline at end of file
+## Running in Docker
+If you want to build and run a docker image of this application you can run:
+```bash
+$ docker build --tag todo-app .
+```
+
+to build the application and then to run it run:
+```bash
+$ docker run todo-app -p 5000
```

Comment:
Did you actually run this command and see it working?

It should be `docker run -p 5000:5000 todo-app`

Docker run options need to be **before** the image name. Anything after it gets passed to the entrypoint.

Also, you should specify both the host port (what port you want to use on your machine) and the container port (what port is the app running on inside the container)